build.py

```diff
@@ -129,7 +129,7 @@ def publish_bintray(source, target, env):
     headers = {
         "Content-type": "application/octet-stream",
         "X-Bintray-Publish": "1",
-        "X-Bintray-Override": "1"
+        "x-override-publish": "1"
```

Comment:
This should be changed in the pax.express backend to match old bintray API. This header change is not really need when its changed at backend side.